apollos-church-api/src/config.js

```diff
@@ -3,8 +3,12 @@ import fetch from 'node-fetch';
 import dotenv from "dotenv/config"; // eslint-disable-line
 import ApollosConfig from '@apollosproject/config';
 
+const configName = process.env.DATABASE_URL
+  ? 'config.postgres.yml'
```

Comment:
I can already tell this isn't going to work. Their content doesn't come from the DB so this is going to select the wrong `config.yml` file

Reply:
Ah, yep. Blocked by [ApollosProject/apollos-templates#449](https://github.com/ApollosProject/apollos-templates/pull/449)